mmdet/core/utils/initialize_utils.py

```diff
@@ -0,0 +1,36 @@
+import torch.optim as optim
+
+
+def caffe2_initialize(model, cfg):
```

Comment:
Using different lr and decay for weight and bias is not specific to caffe2, which has been adopted long ago. The naming is misleading.

Reply:
Add three fileds `bias_lr_mult`, `bias_decay_mult`, `norm_layer_weight_decay` in config files.